doc/generic/pgf/text-en/pgfmanual-en-tikz-matrices.tex

```diff
@@ -477,33 +477,33 @@ \subsubsection{Cell Styles and Options}
 
 In some cases, it is desirable to include some automation in each column/row
 separately. A typical example is to apply stripe-pattern to almost all columns
-and treat only a few of them differently. Then, nesting these keys can open up
+with exceptions. For these type of use-cases, nesting these keys can open up
 a lot of possibilities; in the following example a ``feature comparison'' table
-is demonstrated. It is intentionally made rather verbose to show how the column
-and row settings can be progressively overwritten to create certain effects.
-
-\begin{codeexample}[preamble={\usetikzlibrary{matrix,shapes.misc}}]
-\begin{tikzpicture}
-  [mycol/.style={column #1/.append style={
-     every even row/.style={nodes={fill=gray!10}}}
-   }]
+is demonstrated. It is intentionally made rather verbose and a bit redundant
+to show how the column and row settings can be progressively overwritten to
+create certain effects.
+
+\begin{codeexample}[preamble={\usetikzlibrary{matrix,fit}}]
+\begin{tikzpicture}[
+  font=\sffamily,
+  striped col/.style={column #1/.append style={
+        every even row/.style={nodes={fill=lime!50}}}},
+  head color/.style args={#1/#2}{column #1/.append style={
+        row 1/.append style={nodes={fill=#2}}}}
+]
 
-\matrix [matrix of nodes, nodes in empty cells,
+\matrix [
+   matrix of nodes, nodes in empty cells,
    nodes={text width=2cm, align=center,
           minimum height=1.5em, anchor=center},
-   mycol/.list={1,...,5}, % add mycol style to all cols
-   column 1/.style={
+   striped col/.list={1,...,5}, % add striped col style to all cols
+   column 1/.style={ % Override stripes and modify the feature column
      row 1 column 1/.style={nodes={fill=none, draw=none}},
-     nodes={fill=yellow, chamfered rectangle, inner sep=0,
-            chamfered rectangle corners={north west, south east}},
+     nodes={fill={rgb,256:red,184;green,184;blue,120}, inner ysep=0},
```

Comment:
It this really correct?  `256:red` has a colon but `184;green` and `184;blue` have a semicolon.

Reply:
Yes it's a rather obscure `xcolor` syntax that for some reason people don't use much. That package has a very powerful syntax parser.